### PR TITLE
Add flag for config path

### PR DESCRIPTION
**Pull request by seqizz**
I guess something like this would be pretty useful, since it's not convenient to keep config next to binary at all times.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -29,4 +29,4 @@ go build -a -ldflags="-s -w" -installsuffix cgo -o dist/mouthful main.go
 chmod +x dist/mouthful
 
 # copy over config
-cp ./config.json dist/data/config.json
```
Comment (vkuznecovas) [inline build.sh]: Could we default to dist/data/config.json? We want the sub folder so that we can mount a volume for docker.

File: main.go [hunk 2]
```diff
@@ -16,15 +18,32 @@ import (
 )
 
 func main() {
+
+	configFlag := flag.String("config", "./config.json", "File to read configuration")
```
Comment (vkuznecovas) [inline main.go]: Could we default to ./data/config.json? We want the sub folder so that we can mount a volume for docker.

**Comment by seqizz** [comment]
Pushed again with data :+1:

File: config.json [hunk 3]
```diff
@@ -2,17 +2,17 @@
     "honeypot": false,
     "moderation": {
         "enabled": true,
-        "adminPassword": "somepassword",
```
Comment (vkuznecovas) [inline config.json]: can we leave the default here?

File: config.json [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
+        "adminPassword": "somepaassword",
         "sessionDurationSeconds": 21600,
         "maxCommentLength": 10000,
         "maxAuthorLength": 35
     },
     "database": {
         "dialect": "sqlite3",
-        "database": "./data/test.db"
+        "database": "./test.db"
```
Comment (vkuznecovas) [inline config.json]: "./data/test.db" would be preferable

File: config.json [hunk 5]
```diff
@@ -2,17 +2,17 @@
...
     },
     "api": {
-        "port": 8080,
+        "port": 8780,
```
Comment (vkuznecovas) [inline config.json]: please default to 8080

**Pull request merged**